dome/web_visualization.md

```diff
@@ -0,0 +1,65 @@
+# Overview
+
+Web visualization supports rendering and interacting with running Ignition
+Gazebo simulations. You may find web visualization useful
+when interacting with remote or headless Ignition Gazebo simulations. No
+additionally dependencies, other than a browser and internet connection, are
+required.
+
+## How to use web visualization
+
+The following steps will guide you through the process of running Ignition
+Gazebo with a websocket server and connecting to the websocket for
+visualization. 
+
+1. Start an Ignition Gazebo instance as usual. We will use the `fuel.sdf`
+   world and run it headless.
+```
+ign gazebo -v 4 -s -r fuel.sdf
+```
+
+1. Create an Ignition Launch file with the websocket server plugin.
+```
+echo "<?xml version='1.0'?>
+<ignition version='1.0'>
+  <plugin name='ignition::launch::WebsocketServer'
+          filename='ignition-launch-websocket-server'>
+    <port>9002</port>
+  </plugin>
+</ignition>" > websocket.ign
+```
+
+1. Run the websocket server using
+```
+ign launch -v 4 websocket.ign
+```
+
+1. Visualize the simulation by going to
+   [https://app.ignitionrobotics.org/visualization](https://app.ignitionrobotics.org/visualization) and pressing the Connect button.
```

Comment:
This works for me when I'm in `localhost`. But I tried running from a Docker container, and even if I put the correct IP, I was seeing these errors on the console:

`ERROR DOMException: The operation is insecure.`

Reply:
I think you need `--network host` option. I added a troubleshooting section in 51bf1ff233125ed9994a3beb5a25e2b4c3613f3d.